docs/docs/user-guide.md

```diff
@@ -4,115 +4,141 @@ sidebar_position: 2
 
 # Nullness User Guide (draft)
 
-In Java code, whether an expression may evaluate to null is often documented
-only in natural language, if at all. The goal of JSpecify is to permit
-programmers to express specifications (initially, just nullness properties) in a
-machine-readable way.
+In Java code, whether an expression may evaluate to `null` is often documented
+only in natural language, if at all. JSpecify's nullness annotations let
+programmers express nullness of Java symbols in a consistent and well-defined
```

Comment:
Strictly speaking I think "type usages" is the term.

Reply:
I used "symbols" because I feel like that's a less technical term than "type usages". I mean it to include everything that has a type usage outside of implementation code: class, method, and field declarations. I could change it to "type usages" if we want. I just didn't want to limit it to variables.